cgroup_top.py

```diff
@@ -300,18 +306,23 @@ def collect_ensure_common(data, cgroup):
     data['owner'] = cgroup.owner
     data['type'] = cgroup.type 
 
+def get_user_beacounts():
+    '''
+    get memory stats(via privvmpages) from vzlist output to openvz containers
+    '''
+    prefix = []
+    if os.getuid() != 0:
+       prefix = ['sudo']
+
+    command = prefix + ['vzlist', '-o', 'ctid,privvmpages,privvmpages.l', '-H']
+    return os.popen(" ".join(command)).readlines()
```

Comment:
It broken work with Python 2.6.
check_output was introduced in python 2.7 so won't work for < 2.7, how I see.
With Python 2.6.6 (by default in Centos 6) - 
```
Traceback (most recent call last):
  File "./cgroup_top.py", line 901, in <module>
    main()
  File "./cgroup_top.py", line 875, in main
    collect(measures)
  File "./cgroup_top.py", line 367, in collect
    user_beancounters = get_user_beacounts()
  File "./cgroup_top.py", line 318, in get_user_beacounts
    return subprocess.check_output(command).split('\n')
AttributeError: 'module' object has no attribute 'check_output'
```